### PR TITLE
Major additions, refactoring

### PR DESCRIPTION
**Pull request by FoxNerdSaysMoo**
Added
- Dir importer
- Flake8 workflow
- CODEOWNERS

Updated
- Importers
- Examples
- README

Removed
- AST modification
- Unused dependencies

### CODE REVIEW

**Comment by FoxNerdSaysMoo** [comment]
Added
- Tar support
- Subdirectory option for archives

Updates
- Replaces `requests` with `urllib`

**Comment by GrandMoff100** [comment]
Sweet

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# These owners are the default owners for everything in the repo.
+# Unless a later match takes precedence,
+# These owners will be requested for review when someone opens a pull request.
+* @GrandMoff100
+* @FoxNerdSaysMoo
```
Comment (GrandMoff100) [inline .github/CODEOWNERS:5]: Little ambitious are we?

**Review by GrandMoff100 — CHANGES_REQUESTED** [review]
Workflows need to include black, isort, and pylint. Thanks!

**Review by GrandMoff100 — APPROVED** [review]

**Pull request merged**